.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,47 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
+{
+	"name": "ComfyStream",
+	"build": {
+		"dockerfile": "Dockerfile"
+	},
+	"runArgs": [
+		"--gpus=all",
+		"--workdir", "/workspace"
+	], 
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	// Configure tool-specific properties.
+	"customizations": {
+		"vscode": {
+			"settings": {
+				"python.defaultInterpreterPath": "/workspace/miniconda3/envs/comfystream/bin/python",
+				"python.venvPath": "/workspace/miniconda3/envs",
+				"python.terminal.activateEnvInCurrentTerminal": false,
+				"python.terminal.activateEnvironment": true,
+				"terminal.integrated.shellIntegration.enabled": true
+			},
+			"extensions": [
+				"ms-python.python",
+				"ms-python.black-formatter",
+				"ms-python.vscode-pylance",
+				"ms-python.debugpy"
+			]
+		}
+	},
+	"appPort": [
+		"8188:8188",  // ComfyUI
+		"8889:8889",  // ComfyStream
+		"3000:3000"   // ComfyStream UI (optional)
+	],
+	"forwardPorts": [8188, 8889, 3000],
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// Use 'mounts' to make a list of local folders available inside the container.
+	"workspaceFolder": "/workspace",
+	"workspaceMount": "source=${localWorkspaceFolder},target=/workspace,type=bind",
+	"mounts": [
+		// Use 'mounts' to map to comfyui models on the host
+		"source=${localEnv:SystemDrive}/models/ComfyUI--models,target=/ComfyUI/models,type=bind,consistency=cached",
+		"source=${localEnv:SystemDrive}/models/ComfyUI--models/output,target=/ComfyUI/output,type=bind,consistency=cached"
```

Comment:
WDYT of mounting non-nested folders here? On ai-worker I'm using `./models/ComfyUI--models` and `./models/ComfyUI--output` instead of the nested `./models/ComfyUI-models/output`

Reply:
That's a really good idea! I'll make this change